Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -434,7 +434,8 @@ body {
         }
     }
 
-    &:not(.docked) {
+    /* Only needed by the Mac port, keep in sync with `WI.undockedTitleAreaHeight` */
+    &:is(.mac-platform):not(.docked) {
```

Comment:
i think this should just be `&.mac-platform:not(.docked)`

Reply:
Good thinking, that's simpler. Thanks, I'll update the patch with your suggestion.